azure/extender-servicebus.bicep

```diff
@@ -48,9 +48,17 @@ param skuTier string = 'Standard'
 @description('ISO 8601 Default message timespan to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself.')
 param defaultMessageTimeToLive string = 'P14D'
 
+@description('The tags that will be applied to the resource')
+param tags object = {
+  'radapp.io/environment': context.environment.id
+  'radapp.io/application': context.application.id
+  'radapp.io/resource': context.resource.id
+}
```

Comment:
@AaronCrawfis did you want these to be the names or ids? We used names [here](https://github.com/radius-project/recipes/blob/main/local-dev/rabbitmqqueues.bicep#L60)

For @willvelida the id will be a long string like `/planes/radius/local/..../Applications.Core/applications/my-cool-app`.

The name will be `my-cool-app`.

There's also a special-case to handle where the application can be null. You can see how we handled that here: https://github.com/radius-project/recipes/blob/main/local-dev/rabbitmqqueues.bicep#L60

Reply:
> did you want these to be the names or ids? 

Since these are going to be used to identify an application within the Azure portal and not used for programmatic matching within a namespace, my preference is IDs. That way you can fully identity what's using this resource.

Reply:
@rynowak @AaronCrawfis I've kept the IDs in, but I've added null checking for the application. Hopefully this meets both requirements? 🙂